File: kanye/src/part1/exercise13.rs
// function to return maximum element in an array of numbers
pub fn get_max(arr: &[i32]) -> i32 {
    let mut max = arr[0];
    for i in arr.iter() {
        if i > &max  {
            max = *i;
        }
    }
    println!("The maximum value is {}", max);
    max
}

File: Randy-JRA/Exercise_13.cpp
//Challenge 13 is to Find the maximum number in an array of numbers
#include <iostream>
using namespace std;

int main() {

  int i, n;
  float arr[10]={1,3,2,12,4,5,6,7,8,9};

  // Loop to store largest number to arr[0]
  for(i = 1;i < n; ++i) {

    // Change < to > if you want to find the smallest element
    if(arr[0] < arr[i])
      arr[0] = arr[i];
  }

  cout << endl << "Largest element = " << arr[0];

  return 0;
}

